modules/apps/object/object-info-api/src/main/java/com/liferay/object/info/field/converter/ObjectFieldInfoFieldConverter.java

```diff
@@ -384,8 +383,8 @@ private long _getMaxLength(ObjectField objectField, long defaultMaxLength) {
 	private List<OptionInfoFieldType> _getOptionInfoFieldTypes(
 		ObjectField objectField) {
 
-		String defaultValue = ObjectFieldSettingUtil.getDefaultValueAsString(
-			null, objectField, _objectFieldSettingLocalService, null);
+		String defaultValue = (String)ObjectFieldSettingUtil.getDefaultValue(
+			null, objectField, null);
```

Comment:
Curiosity: Why do you sometimes opt to use the `String.valueOf()` and other cases `(String)`?

Reply:
Done [here](https://github.com/brianchandotcom/liferay-portal/pull/158355/commits/f204fb506a6e0a7e0a43e8cf86c62b3fbdfb5b51)